js/MainPage/CascadeMain.js

```diff
@@ -534,14 +549,14 @@ const loadProject = async () => {
     let fileSystemFile = await file.handle.getFile();
     let jsonContent = await fileSystemFile.text();
     window.history.replaceState({}, 'Cascade Studio','?');
-    initialize(projectContent=jsonContent);
+    new initialize(jsonContent);
```

Comment:
I was confused by this, as `projectContent` is not in scope. So I think I've done the right thing here.